src/Neo/SmartContract/ApplicationEngine.Storage.cs

```diff
@@ -152,7 +153,10 @@ protected internal IIterator Find(StorageContext context, byte[] prefix, FindOpt
             if ((options.HasFlag(FindOptions.PickField0) || options.HasFlag(FindOptions.PickField1)) && !options.HasFlag(FindOptions.DeserializeValues))
                 throw new ArgumentException(null, nameof(options));
             byte[] prefix_key = StorageKey.CreateSearchPrefix(context.Id, prefix);
-            return new StorageIterator(Snapshot.Find(prefix_key).GetEnumerator(), prefix.Length, options);
+            SeekDirection direction = SeekDirection.Forward;
+            if (options.HasFlag(FindOptions.Backwards))
+                direction = SeekDirection.Backward;
```

Comment:
A little syntactic sugar:
```suggestion
            SeekDirection direction = options.HasFlag(FindOptions.Backwards) ? SeekDirection.Backward : SeekDirection.Forward;
```

Reply:
Sure. 3+ years of Go programming and you forget ternary operator exists.